content/100-getting-started/02-setup-prisma/100-start-from-scratch/110-relational-databases/150-using-prisma-migrate.mdx

```diff
@@ -11,7 +11,7 @@ toc: false
 
 <SwitchTech technologies={['*', 'postgresql']}>
 
-In this guide, you'll use [Prisma Migrate](/orm/prisma-migrate) to create the tables in your database. Add the following Prisma data model to your [Prisma schema](/orm/prisma-schema) in `prisma/schema.prisma`:
+In this guide, you'll use [Prisma Migrate](/orm/prisma-migrate) to create the tables in your database. Add the following Prisma ORM data model to your [Prisma schema](/orm/prisma-schema) in `prisma/schema.prisma`:
```

Comment:
Do we really want to call this "Prisma ORM data model"?

Reply:
I believe this is your call at the end of the day. But to me, yes, that makes logical sense and should be what we call it.

Reply:
Actually, I kept it at "Prisma data model" most of the time bc "Prisma ORM data model" sounded weird to me. This might have been one of the few times I changed it. I'm good w doing whatever j lmk

Reply:
Exactly why I pointed it out as well.

I think in this case we can just sidestep this:
```suggestion
In this guide, you'll use [Prisma Migrate](/orm/prisma-migrate) to create the tables in your database. Add the following data model to your [Prisma schema](/orm/prisma-schema) in `prisma/schema.prisma`:
```